### PR TITLE
pcaudiolib 1.2 (new formula)

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

A bit odd to have `espeak-ng` without any audio output so adding `pcaudiolib`.

Available across major repositories - https://repology.org/project/pcaudiolib/versions

May need to skip audit for common flat namespace patch, which we prefer over `autoreconf`-ing

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

File: Formula/e/espeak-ng.rb
Comment (chenrui333) [inline Formula/e/espeak-ng.rb:1]: test failed

File: Formula/p/pcaudiolib.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  head do
+    url "https://github.com/espeak-ng/pcaudiolib.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
```
Comment (chenrui333) [inline Formula/p/pcaudiolib.rb:26]: we can also move this to a general build dep rather than linux-specific

Comment (cho-m) [reply Formula/p/pcaudiolib.rb:26]: It should only be used on linux to find brew's `pulseaudio` and `alsa-lib`, e.g.  https://github.com/espeak-ng/pcaudiolib/blob/master/configure.ac#L34

One macOS, it uses CoreAudio, so doesn't need to look up anything.

File: Formula/e/espeak-ng.rb
Comment (cho-m) [reply Formula/e/espeak-ng.rb:1]: May be new audio output. When I ran test locally, it passed but it now plays voice over speakers. Could be something on CI machine or audio privacy permissions.

Might have to find an option to run to behave like previous test (i.e. no actual audio output).

File: Formula/p/pcaudiolib.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/p/pcaudiolib.rb:26]: 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**